### ISSUE TITLE
Support JSONB value in JOIN query (PostgreSQL)

### ISSUE THREAD

**Issue by sant1ago-da-hanoi** [kdhrubo/db2rest#629]
In a pure PostgreSQL query, I can do it using something like this:

```
select t.value
from tbl_jsonb j
join tbl_translation j
on j ->> key_1 = t.key 
```

So I really hope that DB2Rest supports this kind of action.

**Comment by thadguidry** [comment]
Agree.  And this feels fairly important in our open source version.  JSONB is used for storing records quite often for application use.  And record retrieval via key values is the only way when JSONB is used in PostgreSQL.

Sidenote:  Our other issue for JSON #677 involves bulk inserts, which I'd say is Enterprise only, since there are other tools that can easily be used for bulk uploading records directly to databases where typically bulk loading wouldn't be done through frontend APIs provided like DB2Rest.

**Comment by kdhrubo** [comment]
This issue is already implemented for PGSQL. Needs documentation per discussion on Discord
